assets/dev/scss/frontend/_a-container.scss

```diff
@@ -0,0 +1,6 @@
+.a-con {
+	display: block;
+	height: 120px;
+	padding-block-start: var(--padding-block-start);
+	padding-block-end: var(--padding-block-end);
+}
```

Comment:
I think this should come from the Style Tab.

Reply:
@Omerisra6 ?

Reply:
I think that we will have default styles per Element/Widget type, but I'm not sure when it will be implemented

Reply:
@baghdasarovelementor 

1. Can you check if `display: block;` is coming from the Style Controls by default maybe?
2. Why is the height: 120px? On the FE, even if there is no content it will take at least 120px of space.
3. I'm also not sure about the default padding here. It should also come from the controls?